### ISSUE TITLE
Battle Box Music Timing Not Matching Event's Music Timing

### ISSUE THREAD

**Issue by Duechayapol** [AsoDesu/IslandUtils#3]
Hello!

I am wondering about the Battle Box music that is played in the mod itself, as it starts in the wrong time compared to the event.
Here's what the main event music time goes:
- Pre-game - 20 seconds - music starts at 1:25
- Round countdown - 5 seconds - music at 1:45
- Actual game - 50 seconds - music at 1:50
- Overtime - 10 seconds - overtime music starts (should already be in mcc island)

What I suggest is the following for the mod itself in MCC Island: (timing is based on what is right now on the server)
- Pre-game - 10 seconds (from the round title) - music starts at 1:40
- Actual game - 50 seonds - music at 1:50
- Overtime - 10 seconds - already applied on server

TL;DR music should start at 1:50 to match the event's music timing.

Thank you so much for reading.
Due

**Comment by AsoDesu** [comment]
Hi!
The battle box music was originally started at round start rather than pre-game because Island starts the countdown music.
However i have wanted to change it, just not the highest priority things on my list at the moment.

I will do this when i can :D

**Comment by AsoDesu** [comment]
Added in 1.1.0
